### PR TITLE
Ra Cohen's Introductory Blog Post

### PR DESCRIPTION
**Pull request by MxRaCohen**
An introductory blog post for the latest addition to the EthicalAds team!

### CODE REVIEW

File: content/posts/willful-words-and-pretty-pictures.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+
+[comment]: # (The picture is of kintsugi, a piece of pottery who's cracks are not hidden, but highlighted artfully to tell this plate's story as no other can.)
+
+## About Me
+
+We all tell ourselves stories to live. Some find comfort in narrative, believing events *had* to go a certain way in order to become the person that they are today; others enjoy the *conflict* of the setting, basking in the pure chaos of the universe. As long as there have been people, there have been stories to contextualize our place in history, society, and the world. From cave paintings to modern media, from Homer's Odyssey to Homer Simpson, the storyteller has captured the hearts, minds, and wallets of dreamers and makers alike, for isn't **advertising** another story? One that piques the interest of the passerby, invoking a call to action with the promise of a better tomorrow? 
+
+I'm Ra Cohen, the latest addition to the EthicalAds team. Using pretty pictures, willful words, and data science, I'm on a mission to help people reach their audience and tell their story.
```
Comment (davidfischer) [inline content/posts/willful-words-and-pretty-pictures.md:16]: If possible, try to add a bit of your background here! This is a great place to talk about your machine learning background and say you're a people person.

File: content/posts/willful-words-and-pretty-pictures.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+I'm Ra Cohen, the latest addition to the EthicalAds team. Using pretty pictures, willful words, and data science, I'm on a mission to help people reach their audience and tell their story.    
+
+
+## Pretty Pictures
+
+Humans are visual creatures and it is difficult to overstate the importance of the graphical component of advertising. Guided by the Data-ink ratio, we've found crisp, minimalistic designs, AKA **pretty pictures**, to be most complementary to our publisher's content and thus effective at generating interest. Dedicated to producing excellent outcomes for advertisers and publishers alike, we are happy to assist new advertisers in advertisement creation and to advise experienced advertisers on what types of ads work best on our unique network.
```
Comment (davidfischer) [inline content/posts/willful-words-and-pretty-pictures.md:21]: Visualizing data is definitely an area we haven't excelled at in the past. I think this is a great place to promote your own talents and what your background brings to the group.

File: content/posts/willful-words-and-pretty-pictures.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+## Willful Words
+
+Good writing is the intersection of clarity and purpose. Conveying information and motivating one's audience are best accomplished by the **willful word**, a word mindfully chosen to perform work and left free of clutter. Due to the brevity of advertising, the creatively willful word thrives. The attention of your reader is valuable and respected. We are happy to assist in the creation of your advertising copy upon request. Your time is valuable too, so I endeavor to keep all emails to the point as I assist with advertiser and publisher account management. 
+
+I am also happy to schedule a phone call to discuss your data. 
+
+
+## Data Science
+
+Like all science, the first step to data science is to ask a ***question***. Are you a publisher who wants help understanding how our latest updates affected your profits? Or do you want to compare your data over a few ad placement spots in order to maximise your click through rates? Perhaps you're an advertiser who wants to know how changing your targeting will affect your total number of impressions. Send us an email! I'll crunch the numbers to answer your specific question.
```
Comment (davidfischer) [inline content/posts/willful-words-and-pretty-pictures.md:35]: I want to hear about your data science bonafides!

**Review by davidfischer — COMMENTED** [review]
This is great and I like it a lot. I think dividing up your backgrond unto data science, visualization, and explanation is pretty good. My biggest suggestion will be to talk a bit more about yourself (I know it's hard).

File: content/posts/willful-words-and-pretty-pictures.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+Her father, my greatgrandfather, was a soap salesman himself. One day he was out for a walk by the river early in the morning and he noticed people washing their clothing in the river without bar soap. My greatgrandfather decided then and there to tap that market and would go down on his walk every morning with a box full of soap to sell. Selling by the river early in the morning before the factory opened worked incredibly well until other soap vendors realized the simplistic brilliance of his approach. Soon my greatgrandfather was in competition and learned that he had to get creative with drawing attention. My grandmother cautioned me, "The loud, charasmatic vendor inevitably sells more soap than the quiet, reserved one." It stuck me that this is the mindset that has been shaping advertising for the past 100 years. This is how we have arrived at the current state of the internet, almost unbrowsable without an ad-blocker thanks to pop-ups, banner ads, and booby-trapped x's.
```
Comment (davidfischer) [inline content/posts/willful-words-and-pretty-pictures.md:16]:
```suggestion
Her father, my greatgrandfather, was a soap salesman himself. One day he was out for a walk by the river early in the morning and he noticed people washing their clothing in the river without bar soap. My greatgrandfather decided then and there to tap that market and would go down on his walk every morning with a box full of soap to sell. Selling by the river early in the morning before the factory opened worked incredibly well until other soap vendors realized the simplistic brilliance of his approach. Soon my greatgrandfather was in competition and learned that he had to get creative with drawing attention. My grandmother cautioned me, "The loud, charismatic vendor inevitably sells more soap than the quiet, reserved one." It stuck me that this is the mindset that has been shaping advertising for the past 100 years. This is how we have arrived at the current state of the internet, almost unbrowsable without an ad-blocker thanks to pop-ups, banner ads, and booby-trapped x's. 
```

File: content/posts/willful-words-and-pretty-pictures.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+So I asked my grandma what if yelling was out-lawed? The thought had never occurred to her. I told her about ad-blockers, promising to set her up with one next time I visited, and explained that EthicalAds gets around ad-blockers precisely by being quiet, dignified. My grandma thought for a moment. 
+
+"You have your foot in the door, not a seat at the table. Have you considered making your ads beautiful?"   
+
+I'm Ra Cohen, a data scientist, people person, and the latest addition to the EthicalAds team. Using pretty pictures, willful words, and data science, I'm on a mission to help people reach their audience and tell their story.
```
Comment (davidfischer) [inline content/posts/willful-words-and-pretty-pictures.md:22]: :+1:

File: content/posts/willful-words-and-pretty-pictures.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+Title: Willful Words and Pretty Pictures
+Date: May 25, 2021
+description: Ra's introductory blogpost
```
Comment (davidfischer) [inline content/posts/willful-words-and-pretty-pictures.md:3]:
```suggestion
description: Meet Ra Cohen, the newest addition to the EthicalAds team who will assist our advertisers and publishers in telling their story
```

**Review by davidfischer — APPROVED** [review]
I like this although I made a few small suggestions. I like this better as an introduction to you!

**Comment by davidfischer** [comment]
I think this is just about ready. Shall we update the date on the post and ship it tomorrow or Thursday?

**Comment by MxRaCohen** [comment]
Sounds good to me!

File: content/posts/willful-words-and-pretty-pictures.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
+Title: Willful Words and Pretty Pictures
+Date: May 19, 2021
+description: Meet Ra Cohen, the newest addition to the EthicalAds team who will assist our advertisers and publishers in telling their story
+tags: newsletter, community
```
Comment (ericholscher) [inline content/posts/willful-words-and-pretty-pictures.md:4]:
```suggestion
tags: community, introductions
```

File: content/posts/willful-words-and-pretty-pictures.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+authors: Ra Cohen
+image: /images/posts/2021-ra-cohen-intro.jpg
+image_credit: <span>Photo by <a href="https://unsplash.com/@filmlav?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Fernando Lavin</a> on <a href="https://unsplash.com/s/photos/kintsugi?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>
+
+
+[comment]: # (The picture is of kintsugi, a piece of pottery who's cracks are not hidden, but highlighted artfully to tell this plate's story as no other can.)
+
+## About Me
+
+As I was telling my grandma about my new job, explaining the evils of google ads tracking you around the web and using big flashy banners to distract you from the content that you're actually trying to read, she was first and foremost concerned that someone was personally listening in to every one of her conversations. Once I cleared up that misunderstanding and started describing how EthicalAds doesn't store anyone's data or allow distracting advertising, my grandma expressed some concern.
```
Comment (ericholscher) [inline content/posts/willful-words-and-pretty-pictures.md:14]: -1 on using Google here specifically, unless we have a strong reason to mention them? And if we do, we should capitalize it.

File: content/posts/willful-words-and-pretty-pictures.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+
+
+## Pretty Pictures
+
+As a data scientist, I know just how much impact framing and perspective can have on impressions. A graph zoomed in around all of the action will tell a very different, more turbulent story than the same data presented on a graph with its origin at 0. My design philosophy draws from the Data-Ink ratio principle. The less ink you can use over all, the better, and that ink which you do use should be fundamental to conveying your data. Likewise, I've noticed busy ads with lots of decoration have a lower click through rate than a simple static shot of a logo or a black and white tracing of the product. One of my grandmother's suggestions was actually tracing designs out of coloring books before I gently reminded her that copyright infringement exists. I'm proud to use my eye for artistry to help bring beauty to not only data but also advertising for isn't **advertising** another story? One that piques the interest of the passerby, invoking a call to action with the promise of a better tomorrow?
```
Comment (ericholscher) [inline content/posts/willful-words-and-pretty-pictures.md:27]: > Likewise, I've noticed busy ads with lots of decoration have a lower click through rate than a simple static shot of a logo or a black and white tracing of the product.

Not sure if this is true, in our experience. We generally steer advertisers away from just using the logo, but I don't have full context here.

**Review by ericholscher — APPROVED** [review]
This looks like a good intro post. Definitely a pretty different style than our corporate voice, but I think that's fine in this context.

Just a note for future reviewing -- It's much easier to review writing in GitHub when it has line breaks in it. I recommend [semantic newlines](https://sembr.org/) for more info on this, but doing a standard 80 char break is also good.

File: content/posts/willful-words-and-pretty-pictures.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+
+I am also happy to schedule a phone call to discuss your data. 
+
+
+## Data Science
+
+Like all science, the first step to data science is to ask a ***question***. Are you a publisher who wants help understanding how our latest updates affected your profits? Or do you want to compare your data over a few ad placement spots in order to maximise your click through rates? Perhaps you're an advertiser who wants to know how changing your targeting will affect your total number of impressions. Send us an email; I breathe data! In school, I took courses in our data science graduate program to fulfil my undergrad computer science requirements. After graduation, I went to work as a research scientsit for two years where I built visualization dashboard prototypes for DARPA. I'm glad to now be working at EthicalAds, shaping the future of advertising for a more respectful, beautiful future.
```
Comment (ericholscher) [inline content/posts/willful-words-and-pretty-pictures.md:41]: Love this section. 💯

**Pull request merged**